angular_token_auth.js

```diff
@@ -297,13 +301,19 @@
             }
         };
 
-        //use cookies if available, otherwise try localstorage
-        if (storageMethods['cookie'].test() === true) {
-            return storageMethods['cookie'];
-        } else if (storageMethods['localStorage'].test()) {
-            return storageMethods['localStorage'];
+        if (storageMethods[MODULE_SETTINGS.STORAGE_METHOD]) {
+            return storageMethods[MODULE_SETTINGS.STORAGE_METHOD];
```

Comment:
I don't think we need to run the `test()` method. Those are basically there to help with auto-detection, but that fact they are returning incorrectly is the reason we need to provide an override.